### ISSUE TITLE
Manual logging key error

### ISSUE THREAD

**Issue by C4UT1ON** [jazzband/django-auditlog#614]
Manual logging doesn't seem to work how the documentation suggests it should.

For context, I'm using python 3.11, django 4.2 and django-auditlog 2.3.0

Reproducing this should be as simple as setting up a view action and doing

```
instance = self.get_object()
instance.incident_level = request.data.get("incident_level")
instance.save()
LogEntry.objects.log_create(
            instance=instance,
            action=1,
            changes="{\"incident_level\": instance.incident_level}",
        )
```

i get a KeyError exception because the model is not registered. When i register the model i see my log that i created manually which is great but i also get a duplicate automatic log as a result of registering the model. Its possible that I'm misunderstanding something here but it doesn't seem like using log_create should require the model to be registered and if that is the case, how do i disable the automatic logging of the model once it is registered?

**Comment by hramezani** [comment]
Thanks @C4UT1ON for reporting this.

As you may know, we are working on `V3` major release. it is in beta stage. You can install the `V3 beta` by `pip install django-auditlog --pre`.

Could you please install the beta version and try it again? Please send me the error traceback if it failed.

**Comment by C4UT1ON** [comment]
Updated to v3.0b4
using log_create without registering model crashes with KeyError

Stack Trace:
[django-auditlog KeyError StackTrace.txt](https://github.com/jazzband/django-auditlog/files/14485619/django-auditlog.KeyError.StackTrace.txt)

When i register the model, i still get the duplicate log entries being saved

![image](https://github.com/jazzband/django-auditlog/assets/27019185/d1926001-2c1e-4a92-a803-a88879a823ee)

**Comment by hramezani** [comment]
Thanks @C4UT1ON for preparing the logs. I will try to prepare a fix for this and let you know for testing

**Comment by C4UT1ON** [comment]
I imagine others who are trying to use manual logging might have a similar use case to me so I'm showing a workaround that makes things usable and can be replaced easily when the package is updated.

Essentially my use case is that there is a model where i want all changes to a specific field to be logged along with who changed it and very importantly, why it was changed. If you're trying to log changes to an object manually, most likely you also have control over the saving of the object such that you can do something like:

```
instance.save()
log = LogEntry.objects.get_for_object(instance).order_by("-timestamp").first()
log.additional_data = {"reason": reason}
log.save()
```

i would also strongly suggest looking into whether or not you need to use `transaction.atomic` to ensure the correct log is being updated with the reason if your system could have multiple people editing the instance at one time.

**Comment by hramezani** [comment]
@C4UT1ON based on the log that you sent it seems you are trying to add auditlog for a model that is not registered.

**Comment by C4UT1ON** [comment]
Just to clarify, as it seems there may have been misunderstandings previously. When the model is registered, logs will be created automatically and that's not always the desired behavior. The documentation suggests that manual logging is supported but if i register the model and also try to create my own logs, i will have two set of logs being created per action performed. If its intended for users to have to register models even when they do not require automatic logging, the documentation should include some idea on how to turn off the automatic logging for registered models. I cant consider this issue resolved until either the documentation is updated to show how to turn off automatic logging for registered models or log_create doesn't require the model to be registered

**Comment by hramezani** [comment]
@C4UT1ON sorry for misunderstanding. You are right.
let's keep the issue open

**Comment by hramezani** [comment]
@C4UT1ON I've created https://github.com/jazzband/django-auditlog/pull/627 to fix the problem. Could you please confirm it?

**Comment by hramezani** [comment]
I merged my PR and released v3.0.0.

Please test it with the new version and let me know if the bug still exists.
